unit-tests/py/rspy/timer.py

```diff
@@ -0,0 +1,28 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+import time
+from rspy.stopwatch import Stopwatch
+
+# A timer counting backwards in time(vs forwards in the `stopwatch` class )
+# It supply basic timer API, start, has_expired..
+class Timer:
+
+    _delta = 0
+    _sw = Stopwatch()
+
+    def __init__(self, timeout):
```

Comment:
Not clear what the units are for timeout

Reply:
👍